content/blog/2018-04-23-noo-noo-from-collyers-college-wins-sourcebots-2018.md

```diff
@@ -0,0 +1,80 @@
+---
+title: “Noo Noo” from Collyer's College wins SourceBots 2018
+date: 2018-04-23
+---
+
+SourceBots 2018 reached its exciting climax this weekend as the teams from
+across the south coast gathered at the University of Southampton for the
+competition event. The two-day event saw their robots compete in forty matches
+to determine which would be crowned champion in this year's game
+_Pirate Islands_.
+
+<!--more-->
```

Comment:
This is a marker to Hugo that this is where the "read more" link should be inserted. (I _think_ it needs to be exactly that comment)

Essentially this prevents the intro snippet which is used on the homepage containing the word "Competition" in an odd place (see 33b41a1e55cd341591135ce8f3b11a0e38ae531e).

Reply:
Ah, my apologies. In that case, I'm tempted to suggest documenting the purpose of this comment with another comment (since it's not immediately obvious that this line has a useful effect)

Reply:
No worries. I've only just spotted your reply here (after merging).

I'm not sure about adding another comment, it feels like we'd just be documenting how hugo works, which is in the class of comments I don't really like having.
I don't feel strongly about this, however as this is now merged I'm not going to change it.